packages/ui/SkeletonLoading/index.tsx

```diff
@@ -0,0 +1,22 @@
+import Skeleton from "react-loading-skeleton";
+
+interface SkeletonLoadingProps {
+  width?: number;
+  height?: number;
+  circle?: boolean;
+}
+
+export const SkeletonLoading = ({
+  width,
+  height,
+  circle,
+}: SkeletonLoadingProps) => {
+  return (
+    <Skeleton
+      width={width}
+      height={height}
+      circle={circle}
+      baseColor="#adaeb3"
```

Comment:
Nice touch adding the skeleton! 🚀 

I would change the color to a lighter one, maybe the same as the page's background? Right now it is a bit too heavy in my opinion

```suggestion
      baseColor="#e5e5e5"
```

Reply:
Agreed, the colors could use some adjustments, the grey is IMO too dark and looks too "rough" against the background. But I'm not a color expert so all I Can say is "it looks kinda rough".

Reply:
the scelleton is really cool feature, maybe loose that gradient animation? I like this kind of stuff, like YouTube just some nice grey elements:
![image](https://user-images.githubusercontent.com/72406958/236865538-b03c12f8-8c5c-4280-aa24-e0013eff59b2.png)

Reply:
@PawelPazderski  I agree, just grey elements or maybe a softer gradient would look nice! 👆  Useful thing that comes with gradient is that it moves, so user knows that some data is in loading process and it will be served for him in a moment. ☺️